packages/client/src/views/SysAdmin/Config/Integrations/Integrations.tsx

```diff
@@ -72,6 +169,271 @@ export function IntegrationList() {
+            }
+            onClick={changeModalInfo}
+            type="primary"
+          >
+            {intl.formatMessage(buttonMessages.create)}
+          </Button>
+        ]}
+        autoHeight={true}
+        titleHeightAuto={true}
+        show={showModal}
+        handleClose={toggleModal}
+        title={
+          createClientInfo
+            ? intl.formatMessage(integrationMessages.createClient)
+            : 'Sweet Health'
+        }
+      >
+        {createClientInfo
+          ? intl.formatMessage(integrationMessages.supportingDescription)
+          : intl.formatMessage(integrationMessages.uniqueKeysDescription)}
+
+        {createClientInfo && (
+          <div>
+            <DivSection>
+              <InputField
+                id="name_of_client"
+                touched={false}
+                required={true}
+                label={intl.formatMessage(integrationMessages.name)}
+              >
+                <TextInputStyled
+                  id="client_name"
+                  type="text"
+                  value={clientName}
+                  onChange={onChangeText}
+                  error={false}
+                />
+              </InputField>
+            </DivSection>
+
+            <DivSection>
+              <InputField
+                id="select-input"
+                touched={false}
+                label={intl.formatMessage(integrationMessages.type)}
+              >
+                <SelectOptionStyled
+                  onChange={(val: string) => {
+                    setClientType(val)
+                  }}
+                  value={clientType}
+                  options={[
+                    {
+                      label: intl.formatMessage(
+                        integrationMessages.healthNotification
+                      ),
+                      value: 'health-notification'
+                    },
+                    {
+                      label: intl.formatMessage(integrationMessages.mosip),
+                      value: 'mosip'
+                    },
+                    {
+                      label: intl.formatMessage(
+                        integrationMessages.recordSearch
+                      ),
+                      value: 'record-search'
+                    },
+                    {
+                      label: intl.formatMessage(integrationMessages.webhook),
+                      value: 'webhook'
+                    }
+                  ]}
+                />
+              </InputField>
+            </DivSection>
+
+            {clientType === 'health-notification' && (
+              <AlertSyled type="warning">
+                A notification client (eg. health systems) sends notification of
+                birth and death to OpenCRVS for processing. {'\n'}
+                Please visit{'\n'}
+                <LinkButton
+                  style={{ textAlign: 'left', fontWeight: 'bold' }}
```

Comment:
I think a better alternative for inline `style={{..}}` (applies to all cases where style is used in this file) would be:

```
const AlertLink = styled(Link)`
  text-align: left;
`
```
Link also allows you to add font weight bold with `variant="bold14"` for example.